### PR TITLE
Add text-align property to app lab design elements

### PR DESCRIPTION
**Pull request by pcardune**
Here is what the property row looks like:
screen shot 2016-03-14 at 12 14 56 pm

And here it is hooked up in the code editor:
screen shot 2016-03-14 at 12 50 33 pm

### CODE REVIEW

**Comment by pcardune** [comment]
cc @islemaster

File: apps/src/applab/designElements/EnumPropertyRow.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var rowStyle = require('./rowStyle');
+
+var EnumPropertyRow = React.createClass({
+  propTypes: {
+    initialValue: React.PropTypes.string.isRequired,
+    options: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    handleChange: React.PropTypes.func
```
Comment (islemaster) [inline apps/src/applab/designElements/EnumPropertyRow.jsx]: Your code below assumes that `handleChange` is provided - maybe just make this required for now?

File: apps/src/applab/designElements/EnumPropertyRow.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  handleChange: function (event) {
+    this.props.handleChange(event.target.value);
+    this.setState({selectedValue: event.target.value});
+  },
+
+  render: function () {
+    let options = this.props.options.map(
+      (option, index) => <option key={index} value={option}>{option}</option>
+    );
```
Comment (islemaster) [inline apps/src/applab/designElements/EnumPropertyRow.jsx]: Wish I didn't have to say this, but we're not allowing ES6 yet :frowning: sorry!

**Comment by islemaster** [comment]
Couple of nits, overall this looks really good!

**Comment by pcardune** [comment]
updated with nits. So if es6 isn't allowed, how come it worked at all? Presumably the transforms are being done?

**Comment by islemaster** [comment]
Yes, we are running babelify (to convert JSX) and I think we still have ES2015 turned on.  Yes, it's weird that we support ES6 but don't allow ourselves to write it - I think this is because we're not supporting it in our code-studio package yet.

@Bjvanminnen thoughts on this?  Would it be better/possible to disable the es2015 transform until we decide to support it?

**Comment by Bjvanminnen** [comment]
We support ES6 because it's used heavily in our minecraft stuff (which was initially developed in a sort of separate world). Yes, this is all a little strange :)

**Comment by mehalshah** [comment]
Yay! Anything that means less fiddling to get stuff to align is always great

**Comment by davidsbailey** [comment]
Awesome feature Paul, thanks for helping make design mode more usable!

**Comment by mehalshah** [comment]
One small step for humanity

http://imgh.us/ok_2.svg

**Comment by poorvasingal** [comment]
Just tried it out! This is awesome :D  Thanks Paul!  I've definitely wanted this before when designing my apps.

**Comment by alicesteinglass** [comment]
Love it! Thanks Paul!

**Pull request merged**